network/handlers/drain.go

```diff
@@ -47,7 +49,9 @@ var newTimer = func(d time.Duration) timer {
 }
 
 // Drainer wraps an inner http.Handler to support responding to kubelet
-// probes and KProbes with a "200 OK" until the handler is told to Drain.
+// probes and KProbes until the handler is told to Drain.
+// If the handler is not in Drain status, kubelet probe will get "200 OK" or
```

Comment:
It also seems to be basically a generic additional callback check, i.e. you don't have to restrict it to the kubelet check or anything else, fwiw.

Reply:
done